Dockerfile

```diff
@@ -1,34 +1,32 @@
-FROM centos:7
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
 
-RUN yum -y update && \
-    yum -y install java-1.8.0-openjdk
+LABEL ORG="Armedia LLC" \
+      VERSION="1.0" \
+      IMAGE_SOURCE=https://github.com/ArkCase/ark_base_java8_tomcat9 \
+      MAINTAINER="Armedia LLC"
 
-ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.292.b10-1.el7_9.x86_64
+RUN useradd  --system --user-group tomcat
 
-ENV JRE_HOME /usr/lib/jvm/jre
-
-RUN useradd -r -g root tomcat
-
-
-ADD  http://apache.cs.utah.edu/tomcat/tomcat-9/v9.0.46/bin/apache-tomcat-9.0.46.tar.gz /tmp
+ENV TOMCAT_VERSION=9.0.50  \
+    TOMCAT_MAJOR_VERSION=9
+    
+ADD  https://archive.apache.org/dist/tomcat/tomcat-${TOMCAT_MAJOR_VERSION}/v${TOMCAT_VERSION}/bin/apache-tomcat-${TOMCAT_VERSION}.tar.gz /tmp
 
 RUN cd /tmp && \
-    tar -xf apache-tomcat-9.0.46.tar.gz && \
-    mv apache-tomcat-9.0.46 /opt/tomcat/ && \
-    chown -R tomcat: /opt/tomcat && \
-    sh -c 'chmod +x /opt/tomcat/bin/*.sh'
-
-ENV CATALINA_HOME /opt/tomcat/
-ENV CATALINA_PID /opt/tomcat/temp/tomcat.pid
-ENV PATH=/opt/tomcat/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
-# WORKDIR /opt/tomcat
-USER tomcat
+    tar -xf apache-tomcat-${TOMCAT_VERSION}.tar.gz && \
+    rm apache-tomcat-${TOMCAT_VERSION}.tar.gz &&\
+    cd /tmp && mv apache-tomcat-${TOMCAT_VERSION} /opt/tomcat/ && \
+    chown -R tomcat:tomcat /opt/tomcat && \
+    chmod u+x /opt/tomcat/bin/*.sh && \
+    # Removal of default/unwanted Applications
+    rm -rf /opt/tomcat/webapps/* 
+
+ENV CATALINA_HOME="/opt/tomcat/" \
+    CATALINA_PID="/opt/tomcat/temp/tomcat.pid" \
+    PATH="/opt/tomcat2/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+
+WORKDIR /opt/tomcat2
```

Comment:
What's tomcat2?

Reply:
updated to tomcat.

Reply:
@naveen-armedia  looks like tomcat2 still needs to be updated in workdir above.

Reply:
No @JonHolman with the latest commit I don't see it.  Could you check directly from `tomcat9_pr ` branch